src/main/java/org/fundacionjala/kata/EANValidator.java

```diff
@@ -0,0 +1,20 @@
+package org.fundacionjala.kata;
+
+/**
+ * Class to Validate EAN number
+ */
+public class EANValidator {
+
+    public static boolean validate(String number) {
+        int checksum = 0;
+        int numberValue;
```

Comment:
Remove this unnecessary variable.
you should define the variable in the near scope.